ruby3.2-bouncy-castle-java.yaml

```diff
@@ -0,0 +1,60 @@
+# Generated from http://github.com/jruby/jruby/tree/master/gems/bouncy-castle-java/
+package:
+  name: ruby3.2-bouncy-castle-java
+  version: 1.5.0146.1
+  epoch: 0
+  description: Gem redistribution of "Legion of the Bouncy Castle Java cryptography APIs" jars at http://www.bouncycastle.org/java.html
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - ruby-3.2
+      - ruby-3.2-dev
+  environment:
+    # https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on
+    BCPROV_VERSION: 1.77
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nahi/bouncy-castle-java
+      tag: ${{package.version}}
+      expected-commit: 401f0077cac504c62c449b860366f23393345c35
+
+  - uses: patch
+    with:
+      patches: 0001-add-gemspec.patch
+
+  - runs: |
+      # Bouncycastle does not actively maintain on GitHub so we need
```

Comment:
ACK - so is logstash currently pulling in and using the JDK15 gem which hasn't been updated in many years?

Curious what bumping this to a more recent version may do (not that we've much choice)